cpg_workflows/stages/talos.py

```diff
@@ -16,10 +16,9 @@
 [cohorts.DATASET_NAME.genome]  # repeated for exome if appropriate
 - DATASET_NAME is taken from config[workflow][dataset]
 - historic_results, str, path to historic results directory for this project. If
-  present, new genes and new variant categories will be identified based on the
-  state from the previous run. If absent, panel searching will default to using
-  gene_prior above, and the results from this current run will not be stored in
-  a suitable format to inform the next run.
+  present, new variant categories will be identified based on the state from the
+  previous run. If absent, the results from this current run will not be stored
```

Comment:
???

```suggestion
  previous run. If absent, the results from this current run will be stored
```

Reply:
The author of this PR has made some... interesting changes

Reply:
This specific change is weird, but in the context it makes sense

RE: the `historic results` location in the configuration

"""
historic_results, str, path to historic results directory for this project. If
  present, new variant categories will be identified based on the state from the
  previous run. If absent, the results from this current run will not be stored
  in a suitable format to inform the next run.
"""

Just letting a user know that without specifying a location to save the history file(s), the format required to carry into the next run isn't present

Reply:
Ah go ya. I was reading it so the "if absent" was referring to the file not the argument, ie: "if the history file does not exist, one will be created that can be used in future runs" Perhaps just rephrase it to be explicit.